client/capability-token-2way/capability-token.5.x.rb

```diff
@@ -0,0 +1,19 @@
+require 'twilio-ruby'
+require 'sinatra'
+
+get '/token' do
+  account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+  auth_token = 'your_auth_token'
+  capability = Twilio::JWT::Capability.new(account_sid, auth_token)
+
+  # Create an application sid at
+  # twilio.com/console/phone-numbers/dev-tools/twiml-apps and use it here
+  capability.allow_client_outgoing("APXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
```

Comment:
I would avoid using double quotes and single quotes unless, as Agustin mentioned, we are doing string interpolation.